python/hidet/graph/ops/definitions/conv2d_transpose/conv2d_transpose.py

```diff
@@ -72,29 +74,29 @@ def __init__(
 class Conv2dTransposeOp(Operator):
     def __init__(
         self,
-        x: Tensor,
-        w: Tensor,
+        x: TensorNode,
+        w: TensorNode,
         stride: Tuple[int, int],
         padding: Tuple[int, int, int, int],
         groups: int,
         output_padding: Tuple[int, int],
     ):
         super().__init__(
             inputs=[x, w],
-            attributes={'stride': stride, 'padding': padding, 'groups': groups, 'output_padding': output_padding},
             task=Conv2dTransposeTask(input_like(x, 'x'), input_like(w, 'w'), stride, padding, groups, output_padding),
+            attributes={'stride': stride, 'padding': padding, 'groups': groups, 'output_padding': output_padding},
         )
 
 
 def conv2d_transpose(
     data: Tensor,
     weight: Tensor,
-    stride: Union[int, Sequence[int]],
-    padding: Union[int, Sequence[int]],
+    stride: Union[int, Sequence[int]] = (1, 1),
+    padding: Union[int, Sequence[int]] = (0, 0),
     groups: int = 1,
     output_padding: Union[int, Sequence[int]] = 0,
 ) -> Tensor:
-    sx, sy = normalize_stride(stride)
-    px0, py0, px1, py1 = normalize_padding(padding)
-    opx, opy = normalize_stride(output_padding)  # normalize output padding same as stride
-    return Conv2dTransposeOp(data, weight, (sx, sy), (px0, py0, px1, py1), groups, (opx, opy)).get_output(0)
+    sx, sy = stride  # normalize_stride(stride)
+    px, py = padding  # normalize_padding(padding)
```

Comment:
As a compiler, we need to have a complete support for the padding (in case the upper-framework has this support but we do not). Thus, it is better to keep the original 4 number padding `(px0, py0, px1, py1)` that pads `[h, w]` to `[px0 + h + px1, py0 + w + py1]`.

Reply:
It is better to keep this file unchanged (except the `+ -px0` => `-px0`).